contracts/src/v0.8/ccip/pools/GHO/UpgradeableLockReleaseTokenPool.sol

```diff
@@ -21,6 +21,10 @@ import {IRouter} from "../../interfaces/IRouter.sol";
 /// - Implementation of Initializable to allow upgrades
 /// - Move of allowlist and router definition to initialization stage
 /// - Addition of a bridge limit to regulate the maximum amount of tokens that can be transferred out (burned/locked)
+/// - Modifications from inherited contracts
```

Comment:
maybe better to move the details of `UpgradeableTokenPool` to the contract? and here just say that is based on `UpgradeableTokenPool`, which is an adapted version of `TokenPool`

Reply:
i've still kept the brief change on the target token pool contracts as well, i feel it important the pertinent information is in one place - [327fde2](https://github.com/aave/ccip/pull/18/commits/327fde2701075d57e2e59fd07ba114bf3eaf0a32)